xlators/debug/io-stats/src/io-stats.c

```diff
@@ -1373,19 +1377,21 @@ io_stats_dump_global_to_logfp(xlator_t *this, struct ios_global_stats *stats,
 
     for (i = 0; i < GF_FOP_MAXVALUE; i++) {
         fop_hits = GF_ATOMIC_GET(stats->fop_hits[i]);
-        if (fop_hits && !stats->latency[i].avg)
+        if (fop_hits && !stats->latency[i].avg) {
             ios_log(this, logfp,
                     "%-13s %10" GF_PRI_ATOMIC
                     " %11s "
                     "us %11s us %11s us",
                     gf_fop_list[i], fop_hits, "0", "0", "0");
-        else if (fop_hits && stats->latency[i].avg)
+        } else if (fop_hits && stats->latency[i].avg) {
+            total_fop_hits += fop_hits;
             ios_log(this, logfp,
                     "%-13s %10" GF_PRI_ATOMIC
                     " "
-                    "%11.2lf us %11.2lf us %11.2lf us",
+                    "%11.2lf ns %11.2lf ns %11.2lf ns",
```

Comment:
Do we really need to report ns resolution ? using nanoseconds we'll just increase the shown numbers by 3 digits (you should have also increased the size in the formatting), making them harder to read without providing a real benefit (note that previously we were showing microseconds with 2 decimal places, so the resolution was already in the order of tens of nanoseconds).

Reply:
It is required because we are measuring the time difference in nanoseconds and printing in microsec which is not correct.